app/src/pages/inside/projectSettingsPageContainer/content/testLibrary/testData/datasets/dataset/dataset.scss

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.dataset {
+  &__wrapper {
+    display: flex;
+    align-items: stretch;
+    background-color: $COLOR--bg-000;
+    box-shadow: 0 1px 3px 0 #d5dbe3;
```

Comment:
Pls don't use hex colors.
All colors should be defined in variables. Contact UX team to define the new var if needed.